Flixter/TrailerViewController.h

```diff
@@ -0,0 +1,20 @@
+//
+//  TrailerViewController.h
+//  Flixter
+//
+//  Created by Angelina Zhang on 6/17/22.
+//
+
+#import <UIKit/UIKit.h>
+#import <WebKit/WebKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface TrailerViewController : UIViewController
+
+@property (weak, nonatomic) IBOutlet WKWebView *trailerWebView;
```

Comment:
For some reason it is a convention in UIKit to use hungarian notation for IBOutlets. This is the only time I really see this as an acceptable use case.